test/fixtures/package.json

```diff
@@ -12,7 +12,8 @@
       "exclude": [
         "**/blarg",
         "**/blerg"
-      ]
+      ],
+      "extension": [".es6", ".foo.BAR"]
```

Comment:
Until it is easier to use opts over package.json config, I mixed the casing here a bit to test case-insensitive extension matching.